tests/tt_metal/tt_metal/unit_tests/common/device_fixture.hpp

```diff
@@ -33,7 +33,11 @@ class DeviceFixture : public ::testing::Test {
         for (unsigned int id = 0; id < num_devices_; id++) {
             ids.push_back(id);
         }
-        tt::DevicePool::initialize(ids, 1, DEFAULT_L1_SMALL_SIZE);
+        DispatchCoreType dispatch_core_type = DispatchCoreType::WORKER;
+        if (getenv("TT_METAL_GTEST_ETH_DISPATCH")) {
```

Comment:
can we use something like this  int value = parse_env<int>("TT_METAL_GTEST_ETH_DISPATCH", static_cast<int>(DispatchCoreType::WORKER));

Reply:
Hmm I'm enabling this as a bool flag, not actually taking the enum values. It's the same style as all the other bool envs from metal.